lldb/source/API/CMakeLists.txt

```diff
@@ -145,7 +145,7 @@ add_lldb_library(liblldb SHARED ${option_framework}
   ${option_install_prefix}
 )
 
-# lib/pythonX.Y/dist-packages/lldb/_lldb.so is a symlink to lib/liblldb.so,
+# lib/pythonX.Y/site-packages/lldb/_lldb.so is a symlink to lib/liblldb.so,
```

Comment:
This isn't related to this change, right? This was already the case previously?

Reply:
No it is not. I will revert it.

I looked it up and it turns out that the `dist-` qualification is a Debian rule. I will leave this alone for discussion on another day.